src/pages/Account.vue

```diff
@@ -25,16 +25,16 @@
                             <div class="row">
                                 <div> {{ payment.pended_on }}</div>
                                 <div> {{ payment.vouchers.length }}</div>
-                                <div class="amount">£{{ payment.vouchers.length }}</div>
-                                <div class="email"><i class="fa fa-envelope" aria-hidden="true" title="Send record to my email"></i></div>
+                                <div class="amount">&pound;{{ payment.vouchers.length }}</div>
```

Comment:
I had a look at this the other day and according to some sources using the character is fine as long as you have utf-8 declared in the doc type. I prefer the encoded ones anyway - but would be good to know what the standard is today.